python-checks/src/test/java/org/sonar/python/checks/utils/CodeTestUtils.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.checks.utils;
+
+import org.sonar.api.internal.apachecommons.lang.StringUtils;
+
+public class CodeTestUtils {
+
+  public static String code(String... lines) {
+    return StringUtils.join(lines, '\n');
+  }
```

Comment:
:smile:

Reply:
Still not perfect, but I was inspired by `PythonTestUtils.parse(String... lines)`